### ISSUE TITLE
k=0 included in the first bin, lowering its k value

### ISSUE THREAD

**Issue by eelregit** [bccp/nbodykit#632]
Using `FFTPower`, the first bin k value seems to agree with `k_fundamental * (0 + 6 + 12*np.sqrt(2)) / 19` to 7 digits, implying the title. This may affect other power spectrum code too?

**Comment by rainwoodman** [comment]
Yes. The binning is likely following the numpy convention of using [0, k1). I think other codes do not usually have an incentive to follow numpy convention. I don't think the numpy convention is correct for us, as including k=0 is clearly biasing the estimate of the power in the first bin. Would you file a PR? (Probably shall updated the docstring of FFTPower.

**Comment by eelregit** [comment]
You meant we should switch to `(k_i, k_{i+1}]` in `BinnedStatistic`? I can try that.
This will also affect other modules that use it. Will it break any assumption they make?

**Comment by eelregit** [comment]
poke @rainwoodman

**Comment by rainwoodman** [comment]
I mean only change the first bin to exclude the zero mode -- conceptually this is to always exclude zero mode from binning as the mode contains no information.
